tox.ini

```diff
@@ -4,11 +4,11 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py35, pep8
+envlist = py27, pep8
 
 [testenv]
 usedevelop = True
-commands = py.test
+#commands = py.test
```

Comment:
How about this commented line? Should it be removed?

Reply:
I agree.